salt/_modules/caasp_etcd.py

```diff
@@ -211,6 +220,10 @@ def get_member_id(nodename=None):
                   the local node will be used.
     '''
     command = ["etcdctl"] + get_etcdctl_args() + ["member", "list"]
+    if api_version() == 'etcd2':
+        command.insert(0, "ETCDCTL_API=2")
+    else:
+        command.insert(0, "ETCDCTL_API=3")
```

Comment:
I know it's too late for this, but given that this `command` gets ran directly by `subprocess.check_output` I don't think this is working fine, as the result of the `check_output` command will likely be `OSError: [Errno 2] No such file or directory` as it won't be able to find the `ETCDCTL_API=version` executable in the `PATH`.

We could move the envvar to the end of the command (it should still apply) or use `popen` directly instead.